src/components/charts/SwipeChart.tsx

```diff
@@ -614,6 +617,9 @@ const getStyles = cacheStyles((theme: Theme) => {
       alignSelf: 'flex-start',
       position: 'absolute'
     },
+    cardAdjust: {
+      marginLeft: theme.rem(-1)
+    },
```

Comment:
I don't like this. Normally components should be agnostic to how they are contained. If the chart has built-in margins, those could maybe be reduced to 0 and then move the padding to the container. Otherwise, maybe make the margin configurable, similar to how `marginRem` is passed to buttons. However, `cardAdjust` is too context-specific and hacky. These thinks shouldn't be tied together so closely.

Reply:
Yeah we normally want all components to have their own built in margins, but
this specific situation is the exception since we want it to fill up its card.